content/07.pathogenesis.md

```diff
@@ -372,26 +364,27 @@ In contrast, a model using the prevalence of antibodies among residents of Wuhan
 An estimate of the population of care homes in London found that, among the residents (median age 85), the rate of asymptomatic infection was 43.8%, and among the caretakers (median age 47), the rate was 49.1% [@doi:10.1016/j.eclinm.2020.100533].
 The duration of viral shedding may also be longer in individuals with asymptomatic cases of COVID-19 compared to those who do show symptoms [@doi:10.1038/s41591-020-0965-6].
 As a result, the potential for individuals who do not know they have COVID-19 to spread the virus raises significant concerns.
-In Singapore and Tianjin, two cities studied to estimate incubation period, an estimated 40-50% and 60-80% of cases, respectively, were estimated to be caused by contact with an asymptomatic individuals [@doi:10.7554/eLife.57149].
+In Singapore and Tianjin, two cities studied to estimate incubation period, an estimated 40-50% and 60-80% of cases, respectively, were considered to be caused by contact with asymptomatic individuals [@doi:10.7554/eLife.57149].
 An analysis of viral spread in the Italian town of Vo', which was the site of an early COVID-19 outbreak, revealed that 42.5% of cases were asymptomatic and that the rate was similar across age groups [@doi:10.1038/s41586-020-2488-1].
-They argued that the town's lockdown was imperative for controlling the spread of COVID-19 because it isolated the asymptomatic individuals.
+The argument was thus made that the town's lockdown was imperative for controlling the spread of COVID-19 because it isolated asymptomatic individuals.
 While more models are likely to emerge to better explore the effect of asymptomatic individuals on SARS-CoV-2 transmission, these results suggest that strategies for identifying and containing asymptomatic but contagious individuals are important for managing community spread.
 
 #### Estimating the Fatality Rate
```

Comment:
@SiminaB says:

> Wondering if it would help break up the text a bit to have the IFR and CFR mathematical definitions on separate lines:
> CFR = (number of COVID-19 related deaths)/(number of confirmed or suspected cases)
> IFR = (number of COVID-19 related deaths)/(number of COVID-19 cases)

> This would also emphasize that the numerator is the same.